## moving the <linux/xxxx> includes to below nm includes
## no longer works in 1.0.0.
--- src/platform/wifi/wifi-utils.h
+++ src/platform/wifi/wifi-utils.h.new
@@ -22,10 +22,10 @@
 #ifndef __WIFI_UTILS_H__
 #define __WIFI_UTILS_H__
 
-#include <net/ethernet.h>
 #include <glib.h>
 
 #include "nm-dbus-interface.h"
+#include <linux/if_ether.h>
 
 typedef struct WifiData WifiData;

--- src/settings/nm-settings-connection.h
+++ src/settings/nm-settings-connection.h.new
@@ -22,10 +22,9 @@
 #ifndef __NETWORKMANAGER_SETTINGS_CONNECTION_H__
 #define __NETWORKMANAGER_SETTINGS_CONNECTION_H__
 
-#include <net/ethernet.h>
-
 #include <nm-connection.h>
 #include "nm-types.h"
+#include <linux/if_ether.h>
 
 G_BEGIN_DECLS


